### ISSUE TITLE
Another Content-Type issue with Alpakka

### ISSUE THREAD

**Issue by StefansM** [adobe/S3Mock#210]
Hi, I'm trying to do exactly the same as @sfali in #35. The problem in #35 was addressed in #198 (release 2.1.18), but I'm still seeing a very similar error from Alpakka:

```
Unsupported Content-Type, supported: application/xml; charset=UTF-8
```

After a bit of investigation, I discovered that the `completeMultipartUpload` [method](https://github.com/adobe/S3Mock/blob/master/server/src/main/java/com/adobe/testing/s3mock/FileStoreController.java#L1261) is still returning a `Content-Type` of `application/x-www-form-urlencoded`, which causes Alpakka to choke.

Is this the expected behaviour? I couldn't see a specified `Content-Type` in the S3 docs, but Alpakka is definitely expecting `application/xml`.

Regards,
Stefans

**Comment by agudian** [comment]
@ChaithanyaGK provided a fix, which has been released with version `2.1.19` . @StefansM could you please verify if that fixes this issue?

**Comment by StefansM** [comment]
Yes, that's fixed the issue! Thanks @ChaithanyaGK and @agudian :)
